docs/sphinx/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Is this for windows support ? If so I dont think its required as we currently do not plan to support windows.

Reply:
Yes, the bat file is generated for Windows AFAIK. Let me remove this for the moment. We can regenerate this, if required.